portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,26 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+function displayHobbies() {
+    const title = document.getElementById('title');
+    title.innerText = "This is a list of my main hobbies:";
+    $(function(){
+        $("#panel").load("main_page_clickables/hobbies.html"); 
+    });
+}
+
+function displayAbout() {
+    const title = document.getElementById('title');
+    title.innerText = "Here is some background information about me:";
+    $(function(){
+        $("#panel").load("main_page_clickables/aboutme.html"); 
+    });
+}
+
+function activateListItem() {
+    $('#hobbiesList a').on('click', function (e) {
```

Comment:
I'm not sure this is working right. As far as I know, `.on(..)` registers an event handler. Since this function is already called from a click event, what actually happens is:

- User clicks the `list-group-item`, which as an `onclick=`.
- `activateListItem()` is called
- Every time this function runs, a new `on('click',..)` handler is registered on each `#hobbiesList a`.
- On the second (and nth) click, the callback/event handler is called. 

The call to `.tab('show')` doesn't seem to trigger anything -- I don't think there's a tab to reveal.

Main issues:
 - every click registers _another_ event handler
 - event handler doesn't do anything(?)

It looks like you had plans to make this section more involved, dynamically showing tabs -- that would be really cool.
As it is right now, with just a static list, this could be simplified by using a bulleted list
```
<ul>
 <li>Hobby1..</li>
 <li>..</li>
</ul>
```

Lists are a really commonly used element for this purpose, and are often styled in CSS to not even show bullet points, have borders, etc. It may work well for this too.

Reply:
I intend to print something once each item is clicked. But what I was trying to do first was to set the list item as active once it is clicked. I'm not really sure how to do that, though.

Reply:
Instead of "print", think about "showing"/"hiding" elements that are already present on the page.
A common pattern is this:

 - Add a bunch of items <div class="items item1">...</div><div class="items item2"></div>
 - onclick first hide all items ($("div.items").hide()) and the show the one that matches what the user clicked on ($"(div.item2").show())